specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/examples/EdgeSubscription/ListEdgeSubs.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "subscriptionId": "dd8597b4-8739-4467-8b10-f8679f62bfbf",
+    "resourceGroup": "testResourceGroup",
+    "api-version": "2020-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/dd8597b4-8739-4467-8b10-f8679f62bfbf/resourceGroups/testResourceGroup/providers/Microsoft.AzureStack/edgeSubscriptions/c581d377-070e-4f7c-90f0-e06ea8dca9aa",
+            "name": "c581d377-070e-4f7c-90f0-e06ea8dca9aa",
+            "type": "Microsoft.AzureStack/edgeSubscriptions",
+            "subscriptionId": "dd8597b4-8739-4467-8b10-f8679f62bfbf",
+            "resourceGroup": "testResourceGroup",
```

Comment:
These are not valid top-level resource properties.

Reply:
This also applies to the other examples.

---
In reply to: [439719864](https://github.com/Azure/azure-rest-api-specs/pull/9807#discussion_r439719864) [](ancestors = 439719864)